builtin/providers/openstack/resource_openstack_lb_loadbalancer_v2.go

```diff
@@ -100,6 +108,11 @@ func resourceLoadBalancerV2Create(d *schema.ResourceData, meta interface{}) erro
 		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
 	}
 
+	var lbProvider string
+	if v, ok := d.GetOk("loadbalancer_provider"); ok {
+		lbProvider = v.(string)
+	}
+
```

Comment:
@stack72 I couldn't decide on leaving this part as-is or adding the following:

```go
if v, ok := d.GetOk("provider"); ok && lbProvider != "" {
        lbProvider = v.(string)
}
```

The above would set `lbProvider` if `provider` was set and the new preferred `loadbalancer_provider` wasn't set. However, since having an argument of `provider will error out anyway, is it worth adding the above block of code in?